### DISCUSSION TITLE
Problem while creating apk (cordova)

### DISCUSSION THREAD

**Discussion by antbelogurov** [quasarframework/quasar#10842] [Quasar App Extensions]
[](url)Hi.
I am trying to compile application in cordova following the given instruction.
https://quasar.dev/quasar-cli/developing-cordova-apps/publishing-to-store

1.Created a key It lies in my  C:\Program Files\Java\jdk1.8.0_291\bin
2.Registered the given path to the PATH variable
3.Next, I try to sign the application with this command
`jarsigner -verbose -sigalg SHA1withRSA -digestalg SHA1 -keystore my-release-key.keystore C:\Iot-frontend\src-cordova\platforms\android\app\build\outputs\apk\release\app-release-unsigned.apk alias_name`
[https://i.stack.imgur.com/U4U2K.png](url) 
Invalid keystore format

Passphrase it is not clear what, entered the password from the key. Always reports that the format is incorrect. What am I doing wrong?

**Comment by ontwikkelfabriek** [comment]
@antbelogurov 

Here's a recent thread that could help:
https://stackoverflow.com/questions/66846074/how-to-solve-invalid-keystore-format

btw you'll get better support looking at android/keytool/publish/jarsigner related forums and docs.

**Comment by pikil** [comment]
Try to check -keystore parameter - it should have a proper path
*OR* change the folder you run that command from instead - seems that that `bin` folder might not contain `my-release-key.keystore` file
